toqito/nonlocal_games/binary_constraint_system_game.py

```diff
@@ -0,0 +1,202 @@
+import numpy as np
+import networkx as nx
```

Comment:
[question]: Does `toqito` need a dependency for `networkx` for this to work? It may already be implicit from `numpy`, so maybe it's okay.

Reply:
(This comment is still resolved @Wilson123d )